sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/_statsbeat_metrics.py

```diff
@@ -38,6 +39,18 @@
 
 _HOST_PATTERN = re.compile('^https?://(?:www\\.)?([^/.]+)')
 
+
+class _FEATURE_TYPES:
+    FEATURE = 0
+    INSTRUMENTATION = 1
+
+
+class _StatsbeatFeature:
+    NONE = 0
```

Comment:
Why do we need a NONE feature?

Reply:
Encoding the features enabled into a 64-bit long needs to have a baseline 0 value, which represents all the bits turned off, which represents no features are enabled (which is what NONE represents).